Package.swift

```diff
@@ -27,6 +27,7 @@ let package = Package(
         .Package(url: "https://github.com/IBM-Swift/Kitura.git", majorVersion: 1, minor: 3),
         .Package(url: "https://github.com/IBM-Swift/HeliumLogger.git", majorVersion: 1, minor: 3),
         .Package(url: "https://github.com/IBM-Swift/Kitura-MustacheTemplateEngine.git", majorVersion: 1, minor: 1),
-        .Package(url: "https://github.com/IBM-Swift/Kitura-StencilTemplateEngine.git", majorVersion: 1, minor: 3),
+        .Package(url: "https://github.com/IBM-Swift/Kitura-StencilTemplateEngine.git", majorVersion: 1, minor: 4),
+        .Package(url: "https://github.com/kylef/Stencil", majorVersion: 0, minor: 7)
```

Comment:
> breakage in module A even if no change was made to module A and no breaking change was introduced in module C.
In this case, the version of C in module A's `Package.swift` should be updated and no harm will be caused.

Consider an alternative case where a change in B makes B dependent on a version of C which is incompatible with A. In this case, if you do not have a dependency on C in module A's `Package.swift` you will get a linker error (for example, that some function in C that A expects, is missing.) Instead of getting a clear message that you have versions conflict, you will perform full compilation and get linker error instead, which will be harder to understand. Especially if you compile D that depends on A that depends on B and C. You will have to chase where this linker error came from instead of having clear version conflict error from SPM.